apps/web/src/hooks/useCakePrice.ts

```diff
@@ -9,12 +9,13 @@ import { FAST_INTERVAL } from 'config/constants'
 import { useQuery } from '@tanstack/react-query'
 
 // for migration to bignumber.js to avoid breaking changes
-export const useCakePrice = () => {
+export const useCakePrice = (enabled = true) => {
```

Comment:
same here let's make the parameter an optional object type. For cake we might have chainId as one of the params in the future